Onboarding document (#50)
ConservationMetrics/gc-docs: docs/reference/gc-toolkit/onboarding/index.md

---
sidebar_position: 0
---

# Onboarding

> This document is your onboarding document to a fresh Guardian Connector instance.

Reviewer: Maybe add a meta-note about how this is a template to be used to help onboard a community. So that if someone stumbles onto this page in gc-docs, they have the context to understand its purpose and why it is e.g. written in the first person ("I / you" language).

Could use Docusaurus convention like

```
:::note

...

:::
```

## Introduction

Welcome to your Guardian Connector instance!

This space is your and yours alone, maintained for you by Conservation Metrics Inc.

> Guardian Connector is:
> 
> - where you keep your data safe (a Data Warehouse),
> - tools to get your data into it,
> - tools to use your data for your benefit.

Reviewer: ```suggestion
```

Suggest keeping this out, or linking to a different page, as we already define Guardian Connector elsewhere in the docs, and we want to keep places where we do so to a minimum (so as to avoid updating multiple definitions when it becomes outdated, etc.)

Each instance will have an **alias**. Within the Guardian Connector ecosystem, we refer to your instance through this codeword of your choosing. That adds an extra level of opaqueness on how to access a partner's infrastructure. The idea is for you to name your instance, and can be called however you want.

From now on, we will refer to it as *alias*.

In order to access, you will use its web address: https://alias.guardianconnector.net/

The different tools will be available at https://*.alias.guardianconnector.net/ where * can be one of the services currently available:

- Explorer: https://explorer.alias.guardianconnector.net/ 

> - Files: https://files.alias.guardianconnector.net/ 

Reviewer: ```suggestion
- Filebrowser: https://files.alias.guardianconnector.net/ 
```

- Superset: https://superset.alias.guardianconnector.net/
- Comapeo: https://comapeo.alias.guardianconnector.net/ 
- Windmill: https://windmill.alias.guardianconnector.net/ 

## Authentication

All the services are protected so no one can access them without your permission.

The first barrier is your Alias. Please do not share this with anyone outside of your organization, as it is the first door to your infrastructure.

> We also have a user access list that we will experience in just a second.

Reviewer: ```suggestion
We also utilize user authentication and authorization to only allow approved users to access the resources.
```
(or similar)

> For some services, we might have a predefined user/password combination that I will share with you when we get to them.

Reviewer: ```suggestion
For some services, we use a username and password combination. These will be shared with you.
```

Reviewer: Optional change for the last bit. I see now in the rest of this document that it is written in first-person, and I think it's fine to keep it that way.

## Tools

Guardian Connector is equipped with a suite of tools designed to help communities manage their data and workflows. An extensive walkthrough of these tools is available in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/ 

### Explorer

> It is the service that hosts your Mining alerts, as well as your Comapeo projects. Right now you will only be able to see the mining alerts. Let’s take a look at that.

Reviewer: I would not lead with "it is the service that hosts your Mining alerts." It is much more general purpose than that, and many communities will not have alerts about mining, or alerts at all.

Same logic applies to CoMapeo. 

Can we rephrase this to be generalizable?

Let’s go into https://explorer.alias.guardianconnector.net/ 

#### First time login

You will attempt to login. You can use a gmail account if you have one, or create an account for this service. If you do have a gmail account, it is usually easier to use that one as your login.

Every time the system shows this screen, you will use the same authentication mechanism. It works across the services we provide.

Your access attempt will fail, as you are not authorized yet to access the system. I will give you authorization to access now.

Now you can attempt to access again.

#### Overview

Right now Explorer will only have one view, your current alerts.

You can find more information about Explorer in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/gc-explorer/

#### Whatsapp alerts

#### Ways this space will grow

> As you grow in usage of the system, Explorer will host:
> 
> - Your remote sensing alerts: logging, mining, airstrips and roads
> - Your views of the CoMapeo projects (both maps and media you share within it)

Reviewer: ```suggestion
As you grow in usage of the system, Explorer may host things like:

- Your remote sensing alerts: logging, mining, airstrips and roads
- Your views of mapping projects collected using tools like CoMapeo or KoboToolbox (both maps and media you share within it)
```

#### Customization

The way the map looks like in the alerts dashboard can be customizable.

Let’s discuss what would be relevant for you to have there.

> ### Files
> 
> Files is where you can safely store your files. It is a less structured means for you to preserve your data, whatever it may be.
> Your current access credentials are:

Reviewer: ```suggestion
### Filebrowser

Filebrowser is a tool that you can use to safely store your files. It is a less structured means for you to preserve your data, whatever it may be.
Your current access credentials are:
```

- URL: https://files.alias.guardianconnector.net/ 
- User: 
- Password: 

You can find more information about it in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/filebrowser/

#### Existing data

> You will notice that there are a few folders already available in your Files view, and that more might appear without you doing it.
> This happens because Files is also used for Guardian Connector tools to preserve data that you get from other services.

Reviewer: ```suggestion
You will notice that there are a few folders already available in your Filebrowser view, and that more might appear without you doing it.
This happens because Filebrowser is also used for Guardian Connector tools to preserve data that you get from other services.
```

Change everywhere

Examples of this would be:
- Alerts’ images and data
- Media submissions done through KoboToolbox
- Photos, videos and audios taken with CoMapeo, LocusMaps or other tools

### Superset

> Superset is how we support you to look at your data to get insights out of it.

Reviewer: Can we rephrase this to be a little more technically precise? I know you're trying to avoid jargon-laden language here, but you could also get insights from, say, GC Explorer. So talking about charts etc. will be good here

You can get to it through this web address: https://superset.alias.guardianconnector.net/

As your Data Warehouse grows, you will want to learn more about the patterns in your data. You can find more information about Superset in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/superset/ 

Let’s give you access to it now. It will look a little empty, and that is fine for now. Once you surface the need for it, we will work with you to make good use of it.

### Comapeo

CoMapeo is a collaborative mapping tool that works primarily without internet connection.

> Guardian Connector can preserve your CoMapeo projects through its Archive Server, and make its data available for you through the tools that we’ve seen already.

Reviewer: Here and elsewhere, link to relevant documentation in gc-docs for the tool. We discuss the archive server somewhere already.

Reviewer: i.e. for each of the tools, add a link to the relevant page in gc-docs for more information.

> To sync with Guardian Connector, you will need your CoMapeo Archive Server address:

Reviewer: ```suggestion
To exchange data with Guardian Connector, you will need your CoMapeo Archive Server address:
```

(Awana is moving away from 'sync' language)
